README.md

```diff
@@ -24,27 +23,26 @@ as I don't use Django in my day-to-day Shopify development any more (the last
 version I used with much regularity was Django 1.9) I am not actively working on
 the code.
 
-More recently, @stlk has been actively contributing to the project and now has commit
+More recently, Josef Rousek(@stlk) has been actively contributing to the project and now has commit
```

Comment:
This could be a hyperlink like in this [last section](https://github.com/discolabs/django-shopify-auth#special-thanks).